CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ These are only recommendations (because we will keep backward compatibility for
 #### New PHP events
 
 * Added new event `Db.getTablesInstalled`, plugins should use to register the tables they create.
+* Added new event `Template.confirmPasswordContent`, plugins can inject HTML content to the password confirmation view.
```

Comment:
Cheers for thinking about adding this to the changelog. There's no need though since we don't officially support template events and having it in the changelog might send the wrong message. We won't be simply removing the events and we'll do our best to support them but should the UI or flows change for some reason they might disappear. It's unlikely this will happen in the next few years but wanted to mention it.